### PR TITLE
chore(ci): update fastlane and its plugins

### PR DESCRIPTION
**Pull request by bakoushin**
### Description

Update fastlane and its plugins to the latest version to keep them up to date.

Emerge plugin param `build_type` now is named `tag` (see [changelog](https://github.com/EmergeTools/fastlane-plugin-emerge/blob/77e4484aae38b3c0d98a3a22ff9b9206d691e770/CHANGELOG.md#070)).

```
+--------------------------------------------+
|          Plugin updates available          |
+------------+--------------+----------------+
| Plugin     | Your Version | Latest Version |
+------------+--------------+----------------+
| versioning | 0.4.3        | 0.6.0          |
| emerge     | 0.5.0        | 0.10.4         |
+------------+--------------+----------------+
[19:26:29]: To update all plugins, just run
[19:26:29]: $ bundle exec fastlane update_plugins

#######################################################################
# fastlane 2.222.0 is available. You are on 2.215.1.
# You should use the latest version.
# Please update using `bundle update fastlane`.
#######################################################################
```

### Test plan

* CI
* Nightly build succeeds after the update is merged

### Related issues

NA

### Backwards compatibility

NA

### Network scalability

NA

### CODE REVIEW

**Comment by jeanregisser** [comment]
Small note: the Google Play upload failure will probably be fixed by https://github.com/valora-inc/wallet/pull/5885

**Comment by bakoushin** [comment]
@jeanregisser, the Google Play upload failed for nightly versions because I extended the deadline for the production app. I didn't realize that it should be extended for every app separately. Silas extended for all of them yesterday and it fixed the upload issue.

File: .github/scripts/uploadE2eBuildToEmerge.ts [hunk 1]
```diff
@@ -78,5 +78,5 @@ $.exec(`bundle exec fastlane run emerge \
   base_sha:${baseSha} \
   branch:${branchName} \
   pr_number:${prNumber || ''} \
-  build_type:e2e
+  tag:e2e
```
Comment (bakoushin) [inline .github/scripts/uploadE2eBuildToEmerge.ts:81]: `build_type` now is named `tag` (see [changelog](https://github.com/EmergeTools/fastlane-plugin-emerge/blob/77e4484aae38b3c0d98a3a22ff9b9206d691e770/CHANGELOG.md#070))

**Review by jeanregisser — APPROVED** [review]
Thanks! 🚀

**Pull request merged**